code/obj/item/clown_items.dm

```diff
@@ -65,14 +65,17 @@ VUVUZELA
 			return
 		opened = 1
 		icon_state = "crushed-5"
-		playsound(user.loc, "sound/items/can_open.ogg", 50, 0)
-
+			playsound(user.loc, "sound/items/can_open.ogg", 50, 0)
 		SPAWN_DBG(5)
 			// Wow your joke sucks
-			if(prob(5))
-				playsound(user.loc,"sound/misc/laughter/boo.ogg",50,0)
-			else
-				playsound(user.loc,"sound/misc/laughter/laughtrack[pick("1","2","3","4")].ogg",50,0)
+		if (prob(20))
+			playsound(user.loc, "sound/items/can_open_sienfeld2.ogg", 50, 0)
+		if (prob(10))
+			playsound(user.loc, "sound/items/can_open_sienfeld1.ogg", 50, 0)
+		if (prob(5))
+			playsound(user.loc,"sound/misc/laughter/boo.ogg",50,0)
+		else
+			playsound(user.loc,"sound/misc/laughter/laughtrack[pick("1","2","3","4")].ogg",50,0)
```

Comment:
Previous code is indented into a block of code after the spawn statement, delaying the execution of that whole block.

Current modified version has the spawn be followed just by a series of statements, rather than a code block. Meaning that only the first following statement - the if (prob(20)) condition will be affected and delayed, whereas the rest will execute immediately, including the other condition for a playsound.

Reply:
> Previous code is indented into a block of code after the spawn statement, delaying the execution of that whole block.
> 
> Current modified version has the spawn be followed just by a series of statements, rather than a code block. Meaning that only the first following statement - the if (prob(20)) condition will be affected and delayed, whereas the rest will execute immediately, including the other condition for a playsound.

Thank you for correcting my stuff. I was aware of the maybe ~5% chance multiple play at once but I assumed there was the laughing and bass line playing because it was under else, and i ended up liking the laughing and bass line going over each other.

Regarding the legality... A songs recording is copyrighted work of whoever made it. The Vocal track from Never Gonna Give You Up is the property of Rick Ashley and whoever owns his catalog. But the actual notes and composition is public domain as it is abstract and non-tangible and not specifically associated to the original act.

The baseline in Seinfeld's original recording belongs to Seinfeld however he doesn't own the sound of the Korg M1 nor the concept of playing shakers and light drums to the back of a bass line. My recording of the same bass sound set to custom drums, nothing besides the abstract composition was borrowed and thus, it is legal and fair use.